app/views/orders/_orders.html.erb

```diff
@@ -0,0 +1,31 @@
+
+
+<% if type != "user"%>
+  <h1>Orders Details</h1>
+  <%= render 'order_type_links'%>
+<% end %>
+<table class="table table-striped">
+    <thead>
+      <tr>
+        <th>Order</th>
+        <th>Created_by</th>
+        <th>Created_at</th>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% orders.each do |order|%>
+        <tr>
+          <th><%= link_to "order",single_order_order_path(order[:id])%></th>
+          <% user=User.find_by(id:order[:user_id])%>
```

Comment:
Why are you querying the object in your views.

Reply:
replaced with helper method